src/imageTools/probe.js

```diff
@@ -68,6 +68,18 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneTools) {
         return suv;
     }
     ///////// BEGIN IMAGE RENDERING ///////
+    function pointNearTool(data, coords) {
+        var point = {
+            left: data.handles.end.x,
+            top: data.handles.end.y,
+            width: 0,
+            height: 0
+        };
+
+        var distanceToPoint = cornerstoneMath.rect.distanceToPoint(point, coords);
```

Comment:
Ah yeah sorry I see what you mean. You should just delete the definition of point and use

```` javascript
var distanceToPoint = cornerstoneMath.point.distance(data.handles.end, coords);
````

Try Sublime Text (http://www.sublimetext.com/) for a good editor. It can help with the spacing issue.